examples/Devices/Keyboardio/Model01/Model01.ino

```diff
@@ -86,7 +85,7 @@ static kaleidoscope::plugin::LEDSolidColor solidViolet(70, 0, 60);
 
 const macro_t *macroAction(uint8_t macroIndex, uint8_t keyState) {
   if (macroIndex == 1 && keyToggledOn(keyState)) {
-    KeyboardHardware.serialPort().print("Keyboard.IO keyboard driver v0.00");
```

Comment:
Just on general principle, when you're doing the commit reordering, leave the examples changes to the last commit,so we can know for sure whether we're actually breaking compilation of user sketches?

Reply:
ACK, will do that.